pkg/sql/mutations/mutations.go

```diff
@@ -19,12 +19,16 @@ import (
 )
 
 // StatementMutator defines a func that can change a statement.
-type StatementMutator func(stmt tree.Statement) (changed bool)
+type StatementMutator func(rng *rand.Rand, stmt tree.Statement) (changed bool)
+
+// MultiStatementMutation defines a func that returns additional statements,
+// but must not change any of the statements passed.
+type MultiStatementMutation func(rng *rand.Rand, stmts []tree.Statement) (additional []tree.Statement)
```

Comment:
This interface looks good, but I'm not sure if we should output additional statements or just a new statement list. I don't like the idea of additional statements because it separates like a create table from the alter statements we'd generate for that table. It doesn't have a difference in practice but might make it a little bit harder for people to debug what modifications were made on their tables.

Reply:
These mutations may be used in the sqllogic tests. Currently those print when the column family randomizer changed something, so a user has something to look at in the logs. The whole point of having the return be the new things is exactly so that we can print debug output that shows what mutations were made. This is why I think the current API is better than returning everything.

Reply:
Ok, I agree with that reasoning.